pages/1.13/security/oss/user-account-management/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout.pug
+navigationTitle:  User Account Management
+title: User Account Management
+menuWeight: 10
+excerpt: Managing DC/OS user accounts
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+DC/OS is designed as a multi-user system.
+
+Open DC/OS can handle three types of user accounts:
+
+* **External user accounts**: External user accounts exist for users who want to use their Google, GitHub, or Microsoft credentials. DC/OS does never receive or store the passwords of external users.
+* **Local user accounts**: Local user accounts exist for users who want to create a user account within DC/OS. Usernames and password hashes are stored in CockroachDB on the master nodes.
```

Comment:
Let's not mention CockroachDB here. It's an implementation detail. You can instead say e.g. "in the IAM database" or just "the IAM stores ...".

Reply:
The current "Local user accounts exist for users who want to create a user account within DC/OS." is a little unspecific :-).
 
What do you think about

"Local user accounts are tied to a specific DC/OS cluster. A local user account can be logged into via a so-called service account login or via a username/password-based login"
?